### PR TITLE
Adding TWiRF 11

### PR DESCRIPTION
**Pull request by gnunicorn**
- more tickets
  - marijn's talk
  - how to vote in the berlin elections

### CODE REVIEW

File: _posts/2016-08-19-this-week-in-rustfest-11-more-ticket-special-guest-announcement-and-how-to-vote.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Of course you could sneak away on Sunday, it is a more open format after all. But you might have to travel far to get to your "Wahllokal" to cast your ballot, and who wants to leave this awesome community in the first place? Not to worry though, Germany is a decent representative democracy, so there are ways you can cast your vote without having to go there during RustFest.
+
+The quickest is to just cast your vote _before the elections_ at the "Briefwahlstelle" listed on the bottom of the first page of your notice. Just go there (opening hours listed), bring your ID or passport and you can cast your vote right there and then. If you can't make it there (because you have a job), you can also apply to cast your vote via snail mail. For that you can either a) turn around the notice, tick the first box ("soll on meine obenstehend eingedruckte Anschrift geschickt werden") to have it send to the address listed and sign (name and date) at the fat X, stuff it into an envelope, stamp and mail it. Or b) you fill out this [online form](https://olmera.verwalt-berlin.de/std/IWS/wahlhinweise.do), providing essentially the same information, so they can mail you the ballot and envelopes.
```
Comment (badboy) [inline _posts/2016-08-19-this-week-in-rustfest-11-more-ticket-special-guest-announcement-and-how-to-vote.md]: "soll **an** meine obenstehend ...

File: _posts/2016-08-19-this-week-in-rustfest-11-more-ticket-special-guest-announcement-and-how-to-vote.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+title: "This Week in RustFest 11: More tickets are available, Announcing the our special guest talk and how to vote (in the Berlin elections)"
+socialImageSrc: '/assets/posts/keynote-felix.png'
+authors:
+ - ben
+ - badboy
+---
+
+## More tickets
+
+Since Tuesday 7am (CEST) the second, bigger batch of regular tickets are on sale now. As before, student discount are applicable. We also still have a select few supporter and combo tickets available. The next and final batch of tickets will be announced once these are sold out. Make sure to grab them now.
```
Comment (badboy) [inline _posts/2016-08-19-this-week-in-rustfest-11-more-ticket-special-guest-announcement-and-how-to-vote.md]: the bigger batch **is** on sale now

File: _posts/2016-08-19-this-week-in-rustfest-11-more-ticket-special-guest-announcement-and-how-to-vote.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## Special Guest Talk:The Rust That Could Have Been by Marijn Haverbeke
+
+We are very excited to announce our special guest speaker: Marijn Haverbeke. Many know [Marijn](https://marijnhaverbeke.nl/) as the author of the [Eloquent Javascript](http://eloquentjavascript.net/), [Code Mirror](http://codemirror.net/) and his latest project [Prose Mirror](http://prosemirror.net/). Lesser known is the fact that he has also been involved in the Rust compiler and language during its early days in 2011/2012, Marijn will take us on a journey about features that were in pre-1.0 but were removed – to the better, he argues.
+
+
+[![Marijn's talk](/assets/posts/speaker-marijn.png)](/talks/#the-rust-that-could-have-been)
+
+
+## How to vote (in the Berlin elections)
```
Comment (johannhof) [inline _posts/2016-08-19-this-week-in-rustfest-11-more-ticket-special-guest-announcement-and-how-to-vote.md:24]: Haha awesome

File: _posts/2016-08-19-this-week-in-rustfest-11-more-ticket-special-guest-announcement-and-how-to-vote.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+## More tickets
+
+Since Tuesday 7am (CEST) the second, bigger batch of regular tickets are on sale now. As before, student discount are applicable. We also still have a select few supporter and combo tickets available. The next and final batch of tickets will be announced once these are sold out. Make sure to grab them now.
+
+Further more the application for diversity tickets has closed. Submissions are reviewed while we speak, a member of the team will contact you shortly.
+
+## Special Guest Talk:The Rust That Could Have Been by Marijn Haverbeke
+
+We are very excited to announce our special guest speaker: Marijn Haverbeke. Many know [Marijn](https://marijnhaverbeke.nl/) as the author of the [Eloquent Javascript](http://eloquentjavascript.net/), [Code Mirror](http://codemirror.net/) and his latest project [Prose Mirror](http://prosemirror.net/). Lesser known is the fact that he has also been involved in the Rust compiler and language during its early days in 2011/2012, Marijn will take us on a journey about features that were in pre-1.0 but were removed – to the better, he argues.
```
Comment (johannhof) [inline _posts/2016-08-19-this-week-in-rustfest-11-more-ticket-special-guest-announcement-and-how-to-vote.md]: isn't it _for_ the better?

File: _posts/2016-08-19-this-week-in-rustfest-11-more-ticket-special-guest-announcement-and-how-to-vote.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "This Week in RustFest 11: More tickets are available, Announcing the our special guest talk and how to vote (in the Berlin elections)"
```
Comment (badboy) [inline _posts/2016-08-19-this-week-in-rustfest-11-more-ticket-special-guest-announcement-and-how-to-vote.md]: remove _the_

**Pull request merged**